src/components/FwbListGroup/FwbListGroupItem.vue

```
@@ -34,7 +36,51 @@ const props = defineProps({
+    default: '_self',
+  },
+  tag: {
+    type: String,
+    default: 'li',
+  },
+});
+
+const isLink = computed(() => !!props.href || !!props.to);
+const isRouterLink = computed(() => props.tag === 'router-link' || props.tag === 'nuxt-link');
+const linkComponent = computed(() => {
+  if (isRouterLink) {
```

ESLint

[error] Must use `.value` to read or write the value wrapped by `computed()`.

(vue/no-ref-as-operand)